CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to PlexMediaPlayer
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to PMP. 
+
+#### 1 - Read the Plex CLA
+
+You will need to read the CLA which can be found in our [wiki](https://github.com/plexinc/plex-media-player/wiki/Contributor-License-Agreement#individual-agreement) 
+
+You commit message should include mention to it in the last line ie :
+
+> Plex-CLA-1.0-signed-off-by: Your Name  <your.name@email.com>
+
+#### 2 - General Guidelines
+
+There are the few things that you should check before submitting a PR.
+
+ - Commit messages should be clear enough and start with a topic that they are addressing. they typically should look like : *topic : description*. ie *player : what it fixes* ...
+ - Take time to write a proper commit message description. Most of the time the commit title is not enough to describe the reason why that patch was made. Think of the fact that PMP is running on multiple platforms and that there are other context than your.
+ - Ensure that the patch you're submitting does not break other platforms. i.e. if you are making a patch that is OSX specific it shouldn't break Windows / Linux builds and you should ensure that proper checks are made.
```

Comment:
Change `I.e.` to `For instance,`. Change `OSX specific` to `specific to MacOS, it shouldn't impact Windows or Linux builds.` Remove the rest of the sentence.